api/src/main/java/io/opentelemetry/trace/TraceFlags.java

```diff
@@ -16,45 +16,32 @@
 
 package io.opentelemetry.trace;
 
-import io.opentelemetry.internal.Utils;
-import java.util.Arrays;
-import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
 
 /**
- * A class that represents global trace options. These options are propagated to all child {@link
- * Span spans}. These determine features such as whether a {@code Span} should be traced. It is
- * implemented as a bitmask.
+ * Helper methods for dealing with trace flags options. These options are propagated to all child
+ * {@link Span spans}. These determine features such as whether a {@code Span} should be traced. It
+ * is implemented as a bitmask.
  *
  * @since 0.1.0
  */
 @Immutable
 public final class TraceFlags {
-  // Default options. Nothing set.
-  private static final byte DEFAULT_OPTIONS = 0;
+  private TraceFlags() {}
+
   // Bit to represent whether trace is sampled or not.
   private static final byte IS_SAMPLED = 0x1;
 
   private static final int SIZE = 1;
   private static final int BASE16_SIZE = 2 * SIZE;
-  private static final TraceFlags DEFAULT = fromByte(DEFAULT_OPTIONS);
-
-  // The set of enabled features is determined by all the enabled bits.
-  private final byte options;
-
-  // Creates a new {@code TraceFlags} with the given options.
-  private TraceFlags(byte options) {
-    this.options = options;
-  }
 
   /**
-   * Returns the size in bytes of the {@code TraceFlags}.
+   * Returns the size in Hex of trace flags.
    *
-   * @return the size in bytes of the {@code TraceFlags}.
-   * @since 0.1.0
+   * @since 0.9.0
    */
-  public static int getSize() {
-    return SIZE;
+  public static int getBase16Size() {
```

Comment:
Think we're using `getHexLength`

https://github.com/open-telemetry/opentelemetry-java/blob/master/api/src/main/java/io/opentelemetry/trace/TraceId.java

Reply:
yep. good catch.

Reply:
done